src/applications/vaos/services/appointment/index.js

```diff
@@ -227,25 +226,15 @@ export function getPatientEmail(appointment) {
 }
 
 /**
- * Filter method for future confirmed appointments
- * @param {Object} appt A FHIR appointment resource
- */
-export function filterFutureConfirmedAppointments(appt) {
-  const apptDateTime = moment(appt.start);
-
-  return (
-    !appt.vaos.isPastAppointment &&
-    !FUTURE_APPOINTMENTS_HIDDEN_SET.has(appt.description) &&
-    apptDateTime.isValid() &&
-    apptDateTime.isBefore(moment().add(13, 'months'))
-  );
-}
-
-/**
- * Filter method for past appointments
+ * Checks to see if a past appointment has a valid status and is in
+ * the given date range
+ *
  * @param {Object} appt A FHIR appointment resource
+ * @param {Object} startDate A Moment date object containing the start date of the range
+ * @param {Object} endDate A Moment date object containing the end date of the range
+ * @returns Whether or not the appt should be shown
  */
-export function filterPastAppointments(appt, startDate, endDate) {
+export function isValidPastAppointment(appt, startDate, endDate) {
```

Comment:
I renamed and consolidated some functions in here around filtering, to make it more clear what they're doing.